install/kubernetes/cilium/values.yaml

```diff
@@ -1580,3 +1580,8 @@ clustermesh:
 externalWorkloads:
   # -- Enable support for external workloads, such as VMs (false by default).
   enabled: false
+
+# -- Configure BGP
+bgp:
+  # -- Enable announcement of service LoadBalancer IPs
+  lbIP: false
```

Comment:
It's a bit tricky with BGP because `enabled` alone doesn't mean anything. We need to know if the user is interested in LB IP announcements or PodCIDR announcements (coming soon), etc. Hence why there's `lbIP`. Any ideas?

Reply:
@christarazi what about:
```
bgp:
   enabled: true|false
```
does the user needs to know about `lbIP`?

Reply:
But what does the following mean?

```
bgp:
   enabled: true
   announce:
      lbIP: false
      podCIDR: false
```

Reply:
Fair enought, will implement.

Reply:
Done